Extension-React/public/manifest.json

```diff
@@ -5,8 +5,7 @@
     "version": "0.1",
     "permissions": ["activeTab", "storage"],
     "background": {
-        "service_worker": "background.js",
-        "scripts": ["background.js"]
```

Comment:
We need this "scripts" for it to work on firefox

Reply:
fixed with 2c871e8ce3e96742da6f2e5f1f3206c5e69ad397